static/admin/config.yml

```
@@ -119,23 +119,23 @@ collections:
     editor:
       preview: false
     fields:
-      - { label: "Title", name: "resume-page-title", widget: "string" }
+      - { label: "Title", name: "resume-page-title", widget: "string",pattern: ['.{15}', "Must have at least 15 characters"] }
       - label: "Work experience"
         name: "work-experience"
         widget: "list"
         fields:
-          - { label: "Job title", name: "job-title", widget: "string" }
-          - { label: "Company name", name: "company-name", widget: "string" }
-          - { label: "Job description", name: "job-description", widget: "string" }
+          - { label: "Job title", name: "job-title", widget: "string",pattern: ['.{3,50}', "Must have 3-50 characters"] }
```

```suggestion
          - { label: "Job title", name: "job-title", widget: "string", pattern: ['.{3,50}', "Must have 3-50 characters"] }
```
please add a trailing space before pattern and repeat in every occurrence 😄